src/rest/obsRest.js

```diff
@@ -15,8 +15,8 @@ const api = {
 
   fetchObsByPatient: (patient, concepts, answers, groupingConcepts, limit) => {
     return axiosInstance.get(`obs/?patient=${patient}&concepts=${concepts.constructor === String ? concepts : concepts.join(",")}
-      ${answers.constructor === String ? `&answers=${answers}` : (answers.length > 0 ? `&answers=${answers.join(",")}` : '')}
-      ${groupingConcepts.constructor === String ? `&groupingConcepts=${groupingConcepts}` : (groupingConcepts.length > 0 ? `&groupingConcepts=${groupingConcepts.join(",")}` : '')}
+      ${answers !== null ? (answers.constructor === String ? `&answers=${answers}` : (answers.length > 0 ? `&answers=${answers.join(",")}` : '')) : ''}
```

Comment:
@mogoodrich How do I test this, I did by just checking if the summary UI was still displaying properly, not sure how dave knew it was broken.

Reply:
@jomadoye strangely, it wasn't failing for all patients... let me see if Dave can confirm an example patient...

Reply:
@jomadoye this still might fail, though, if the answer and groupConcepts parameters are not passed in?